web-ui/src/main/resources/catalog/views/default/templates/recordView/featurecatalogue.html

```diff
@@ -1,7 +1,34 @@
-<div data-ng-repeat="featureType in mdView.current.record.featureTypes">
-  <h3 data-translate="">featureAttributeTable</h3>
+<div
+  class="panel panel-default gn-margin-top width-50"
+  data-ng-show="mdView.current.record.featureTypes.length > 0"
+>
+  <div class="panel-heading" data-gn-slide-toggle="true">
+    <span class="text-uppercase" data-translate="">tableOfContents</span>
+  </div>
+  <div class="panel-body">
+    <ul class="list-group">
+      <li
+        class="list-group-item"
+        data-ng-repeat="featureType in mdView.current.record.featureTypes"
```

Comment:
```suggestion
        data-ng-repeat="featureType in mdView.current.record.featureTypes"
        data-ng-if="featureType.typeName !== ''"
```

Filter empty ones ?
It should not happen usually but just in case.

![image](https://user-images.githubusercontent.com/1701393/203616690-64a30d80-981c-4a03-93ee-14524c2adfbf.png)

Reply:
Good question. Haven't thought about it because I've never encountered metadata with these amounts of tables before

Reply:
We can check later about this.